web/html/src/components/input/Select.js

```diff
@@ -36,32 +47,49 @@ type Props = {
 export function Select(props: Props) {
   const {
     inputClass,
-    children,
+    options,
+    getOptionLabel,
+    getOptionValue,
+    formatOptionLabel,
+    placeholder,
+    isClearable,
     ...propsToPass
   } = props;
   const formContext = React.useContext(FormContext);
+  const convertedOptions = options instanceof Array && options.every(item => !(item instanceof Object)) ?
```

Comment:
I think this could be simplified a bit, right now we loop over `options` twice. Perhaps something along these lines?
```ts
const convertedOptions = (options || []).map(item => typeof item === 'string' ? { label: item, value: item } : item);
```
Unless there's an additional edge case I missed here and options can also be something that isn't an array at all?

Reply:
indeed

Reply:
done